addons/l10n_mx/models/chart_template.py

```diff
@@ -0,0 +1,103 @@
+# coding: utf-8
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, api, _
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    @api.multi
+    def _load_template(
+            self, company, code_digits=None, transfer_account_id=None,
+            account_ref=None, taxes_ref=None):
+        self.ensure_one()
+        accounts, taxes = super(AccountChartTemplate, self)._load_template(
+            company, code_digits=code_digits,
+            transfer_account_id=transfer_account_id, account_ref=account_ref,
+            taxes_ref=taxes_ref)
+        account_tax_obj = self.env['account.tax']
+        account_obj = self.env['account.account']
+        # Due to the fact that the template does not have the fields
+        # 'use_cash_basis' and 'cash_basis_account' we are making this ugly
+        # hack which should be removed in master once those fields are add in
+        # account.tax.template
+        taxes_acc = {
+            'ITAX_010-IN': account_obj.search([('code', '=', '208.01.01')]),
+            'ITAX_160-IN': account_obj.search([('code', '=', '208.01.01')]),
+            'ITAXR_04-OUT': account_obj.search([('code', '=', '216.13.01')]),
+            'ITAXR_10-OUT': account_obj.search([('code', '=', '216.13.01')]),
+            'ITAX_1067-OUT': account_obj.search([('code', '=', '216.13.01')]),
+            'ITAX_167-OUT': account_obj.search([('code', '=', '216.13.01')]),
+            'ITAX_010-OUT': account_obj.search([('code', '=', '208.01.01')]),
+            'ITAX_160-OUT': account_obj.search([('code', '=', '208.01.01')])}
+
+        for tax in self.tax_template_ids:
+            if tax.description not in taxes_acc:
+                continue
+            account_tax_obj.browse(taxes.get(tax.id)).write({
+                'use_cash_basis': True,
+                'cash_basis_account': taxes_acc.get(tax.description).id,
+            })
+        return accounts, taxes
+
+    @api.multi
+    def generate_account(
+            self, tax_template_ref, acc_template_ref, code_digits, company):
+        res = super(AccountChartTemplate, self).generate_account(
+            tax_template_ref, acc_template_ref, code_digits, company)
+        self.env['account.account'].browse(res.values()).assign_account_tag()
```

Comment:
You shouldn't do this unless the localization you're installing is Mexican one. Please check the account chart you are installing before.